packages/app/src/elements/org-settings.ts

```diff
@@ -116,6 +119,57 @@ export class OrgSettingsView extends Routing(StateMixin(LitElement)) {
         }
     }
 
+    private async _enableDirectorySync() {
+        const confirmed = await confirm(
+            $l(
+                "Do you want to enable Directory Sync via SCIM for this organization? You will be given a unique URL to provide to your Active Directory or LDAP server for synchronizing and provisioning members."
+            ),
+            $l("Confirm")
+        );
+
+        if (!confirmed) {
+            return;
+        }
+
+        // TODO: In the future, ask if only groups/members should be synchronized
+
+        await app.updateOrg(this._org!.id, async (org) => {
+            org.directory.syncProvider = "scim";
+            org.directory.syncGroups = true;
+            org.directory.syncMembers = true;
+            org.directory.scim = new ScimSettings();
+            org.directory.scim.secret = stringToBytes(await uuid());
+        });
+
+        await app.synchronize();
```

Comment:
@MaKleSoft this isn't working for me, do you know why? The org keeps the previous/default `directory` values.

Reply:
You don't need `synchronise`, since `app.updateOrg` already takes care of that. What you'll need to do is to update the `updateOrg` endpoint  here: https://github.com/padloc/padloc/blob/v4/packages/core/src/server.ts#L915